package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "react-native-tab-view": "0.0.57",
-    "react-navigation": "1.0.0-beta.7"
+    "react-navigation": "1.0.0-beta.9"
```

Comment:
Please, remove this changes. ```react-navigation@1.0.0-beta.9``` will be support in [next branch](https://github.com/LeoLeBras/react-router-navigation/tree/next)